src/Administration/Resources/app/administration/src/app/component/form/sw-custom-field-set-renderer/index.js

```diff
@@ -149,6 +149,12 @@ Component.register('sw-custom-field-set-renderer', {
     },
 
     watch: {
+        'entity.customFields': {
```

Comment:
I would prefer to set a key value in `platform/src/Administration/Resources/app/administration/src/module/sw-category/view/sw-category-detail-base/sw-category-detail-base.html.twig`. This forces a rerender of the component and does not have side effects

(More information: https://michaelnthiessen.com/force-re-render/#the-best-way-key-changing)